jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNode.kt

```diff
@@ -193,22 +194,34 @@ class AwsExplorerLoadingNode(project: Project) :
 }
 
 class AwsExplorerErrorNode(project: Project, exception: Exception) :
-        AwsExplorerNode<Exception>(project, exception, null) {
+    AwsExplorerNode<Exception>(project, exception, null) {
 
     override fun getChildren(): Collection<AbstractTreeNode<Any>> = emptyList()
 
     override fun update(presentation: PresentationData) {
         presentation.apply {
             // If we don't have a message, at least give them the error type
             tooltip = value.message ?: value.javaClass.simpleName
-            addText(MSG, SimpleTextAttributes.ERROR_ATTRIBUTES)
+
+            val exception = value
+            val errorDetails = if (exception is AwsServiceException) {
+                val awsErrorDetails = exception.awsErrorDetails()
+                "${awsErrorDetails.serviceName()} | ${awsErrorDetails.errorCode()}"
```

Comment:
Not sure how I feel about ` | `, what about `: `. No space before, only after

Reply:
Coming up
![image](https://user-images.githubusercontent.com/39839589/49193449-f1c68100-f333-11e8-8d5f-5f0d19fc8277.png)

Reply:
Me likely.